app/models/behaviour.rb

```diff
@@ -0,0 +1,49 @@
+class Behaviour < ApplicationRecord
+  belongs_to :bot
+
+  validates :name, presence: true
+  validates :kind, inclusion: { in: %w(front_desk) }
+
+  validate :config_must_match_schema
+
+  default_scope { order(:order) }
+
+  def self.create_front_desk!(params = {})
+    default_params = {
+      kind: "front_desk",
+      name: "Front Desk",
+      order: 0,
+      enabled: true,
+      config: {
+        "greeting" => { "message" => { "en" => "" }},
```

Comment:
Config should be without `greeting.message.en`. either `greeting.message` or just `greeting`.
And leave assume the config is for the main language implicitly.